domains/firefly.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vibred",
+        "email": "vibred@pm.me"
+    },
+    "record": {
+        "NS": ["cora.ns.cloudflare.com", "jonah.ns.cloudflare.com"]
```

Comment:
CI doesnt support NS records at the moment

Reply:
> CI heyhey... I am in the waiting list! 我只是在排队

Reply:
NS will probably be used for maintainers and trusted users, but im not sure yet

Reply:
> heyhey... I just join in the waiting list ! See if Andrew let us use NS record

This is is-a.dev not Open Domains mate

Reply:
I know,,by the side, can you help me check the request code here?
https://github.com/open-domains/register/pull/1164/commits
O,O